lib/index.js

```diff
@@ -60,7 +60,7 @@ exports.register = function (plugin, options, pluginNext) {
 
     const cssBaseUrl = `${settings.endpoint === '/' ? '' : settings.endpoint}/css`;
 
-    plugin.dependency(['inert', 'vision'], (server, serverNext) => {
+    server.dependency(['inert', 'vision'], () => {
```

Comment:
I just found out that this might not be a good pattern. This is a callback, it's not guaranteed to happen on the same tick, so I guess we need to promisify this.

Reply:
It may be patchable in hapi itself to make it promise-ready, need to investigate.

Reply:
Or not... You can use node's utility to promisify that thing.

Reply:
I moved it out to exports instead, pretty sure that was added in Hapi 17

Reply:
Except it's not the same thing. Hapi doesn't guarantee you those deps will be loaded when the register function is run.

Reply:
Very true!  This function is called during server init, not during plugin registration.  If you want to run this code during plugin registration, one approach is to make vision and inert peer dependencies then register them directly inside lout (since those plugins are allowed to be registered multiple times).  Another option would be to assert that they've already been registered (`e.g. Hoek.assert(server.registrations.vision)`), but this option is the only one that will force users to do the proper plugin registration order, which usually causes confusion.